test/spec/java_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe "Java" do
 
-  ["1.7", "1.8", "8", "1.9", "9", "9.0.0", "10", "11", "12", "13", "14",
+  ["1.7", "1.8", "8", "11", "13", "14", "15",
```

Comment:
It looks like testing against "1.9", "9", "9.0.0", "10" is being dropped here.
Could you add the reasoning to the PR description? :-)